src/components/comments/CommentActions.tsx

```diff
@@ -69,13 +74,17 @@ export const CommentActions: React.FC<CommentActionsProps> = ({
   const { isOpen, toggleOpen, wrapperRef } = useDropdown()
 
   return (
-    <>
+    <Container>
       {isResolveEnabled && (
         <CommentResolveButton comment={comment} onClick={toggleResolve} />
       )}
       {isActionsEnabled && (
         <DropdownContainer ref={wrapperRef}>
```

Comment:
should the `DropdownList` be smaller similar to the design

Reply:
Done